error TS2304: Cannot find name 'TransformStream'  'ReadableStream' 'WritableStream'

Since `9.0.1`, when i do `npm install`:

````
node_modules/execa/types/stdio/type.d.ts:56:52 - error TS2304: Cannot find name 'TransformStream'.

56  | Unless<IsSync, DuplexTransform | WebTransform | TransformStream>;
                                                      ~~~~~~~~~~~~~~~

node_modules/execa/types/stdio/type.d.ts:78:63 - error TS2304: Cannot find name 'ReadableStream'.

78  | Unless<IsSync, (AsyncIterable<unknown> & ProcessStdinFd) | ReadableStream>;
                                                                 ~~~~~~~~~~~~~~

node_modules/execa/types/stdio/type.d.ts:88:19 - error TS2304: Cannot find name 'WritableStream'.

88  | Unless<IsSync, WritableStream>;
                     ~~~~~~~~~~~~~~

node_modules/execa/types/transform/normalize.d.ts:54:22 - error TS2304: Cannot find name 'TransformStream'.

54  readonly transform: TransformStream;
                        ~~~~~~~~~~~~~~~


Found 4 errors in 2 files.

Errors  Files
     3  node_modules/execa/types/stdio/type.d.ts:56
     1  node_modules/execa/types/transform/normalize.d.ts:54
```

An idea? Thanks a lot

--- a/test-d/stdio/option/readable-stream.test-d.ts
+++ b/test-d/stdio/option/readable-stream.test-d.ts
@@ -1,3 +1,4 @@
+import {ReadableStream} from 'node:stream/web';
 import {expectError, expectAssignable, expectNotAssignable} from 'tsd';
 import {
 	execa,

--- a/test-d/stdio/option/writable-stream.test-d.ts
+++ b/test-d/stdio/option/writable-stream.test-d.ts
@@ -1,3 +1,4 @@
+import {WritableStream} from 'node:stream/web';
 import {expectError, expectAssignable, expectNotAssignable} from 'tsd';
 import {
 	execa,

--- a/types/stdio/type.d.ts
+++ b/types/stdio/type.d.ts
@@ -1,4 +1,5 @@
 import type {Readable, Writable} from 'node:stream';
+import type {ReadableStream, WritableStream} from 'node:stream/web';
 import type {
 	Not,
 	And,

